packages/cardano-services/src/Utxo/DbSyncUtxoProvider/util.ts

```diff
@@ -0,0 +1,5 @@
+import { Cardano } from '@cardano-sdk/core';
+
+export const generateAssetId = (policy: string, name: string) => Cardano.AssetId(policy + name);
+
+export const generateUtxoId = (txHash: string, index: number) => `${txHash}:${index}`;
```

Comment:
```suggestion
export const createUtxoId = (txHash: string, index: number) => `${txHash}:${index}`;
```

+ move to `core/src/Cardano/util/utxo.ts`